.github/workflows/deploy.yml

```diff
@@ -21,6 +21,7 @@ permissions:
 
 jobs:
   dependency-review:
+    if: github.event_name == 'pull_request' || github.event_name == 'pull_request_target' || github.event_name == 'merge_group'
```

Comment:
Sorry hadn't spotted this in the original PR - the `dependency-review` action shouldn't be in the `deploy.yml` workflow, it should be it's own workflow that runs on PR raised. As it's scanning the dependencies and ensuring only safe code is added.

So if you remove `dependency-review` you won't need this `if` conditional